cc/trees/layer_tree_host.cc

```diff
@@ -1660,6 +1663,9 @@ void LayerTreeHost::RemoveSurfaceRange(const viz::SurfaceRange& surface_range) {
 }
 
 void LayerTreeHost::AddLayerShouldPushProperties(Layer* layer) {
+  recordreplay::Assert(
+      "[Run-1229-1342] LayerTreeHost::AddLayerShouldPushProperties %d",
```

Comment:
same here

Reply:
Also, should we also add the frame id to the assertion as well (or are `layer` ids unique amongst all frames)?

Reply:
Layer ids are globally unique. But we can add the frame id if it gives us additional context.